lib/message.js

```diff
@@ -20,20 +20,20 @@ const { camelCase, get, invoke } = require('lodash');
 
 const paths = {
   browser: ({ suite }) => ['suite', 'runner', suite.cid, 'browserName'],
-  details: () => 'suite.err.stack',
+  details: () => 'suite.error.stack',
   flowId: ({ suite }) => ['reporter', 'baseReporter', 'stats', 'runners', suite.cid, 'sessionID'],
```

Comment:
this needs to be updated. I fixed using:
```
flowId: () => 'reporter.browser.sessionId',
```

This may also work, based on the console.log output of the ctx object, but I haven't tested
```
flowId: () => 'reporter.driver.sessionId',
```